src/utils.ts

```diff
@@ -122,11 +122,11 @@ export async function getMessagesByUserIn(client: MatrixClient, sender: string,
         }
     }
 
-    function initialSync() {
-        const qs = {
-            filter: JSON.stringify(filter),
-        };
-        return client.doRequest("GET", "/_matrix/client/r0/sync", qs);
+    // There is no direct replacement for this and doing a sync here
```

Comment:
This is not a function documentation but implementation comment, so this should go inside the function.

Also, it's not clear what you're commenting. Let me suggest:
```js
/*
  Note: `/initialSync` is deprecated. However, there is no replacement for this API for the time being.
  While previous versions of this function used `/sync`, experience shows that it can grow extremely
  slow (4-5 minutes long) when we need to sync many large rooms, which leads to timeouts and
  breakage in Mjolnir, see e.g. [insert issue numbers].
*/
```